### ISSUE TITLE
Draft Data Protection Principles

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#288]
See the OMF [State of the Practice Wiki](https://github.com/openmobilityfoundation/privacy-committee/wiki/Mobility-Data-Management-State-of-Practice) for reference materials from other cities.

**Comment by amenity** [comment]
@johnclary is this happening? Backlog? On Deck for Sprint 4?

**Comment by johnclary** [comment]
No. Rolling this into Micrmobiltiy Privacy. Tagged as such and moving to backlog. Thanks.

**Comment by johnclary** [comment]
I reached out to Karla to see if we could get Chris's help on this. Now that we have [draft guidelines](https://docs.google.com/document/d/1JvVSWw1-VwFdYqQIefvKvM0RmfEK2tv1wTyjeyNUavY/edit) from the OMF. I'd like to start working on ours. Slated for Sprint 26.

**Comment by amenity** [comment]
@johnclary was this covered by any of your recent OMF hustle?

**Comment by amenity** [comment]
@johnclary - Sprint 33?
